<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBC</class>
 <widget class="QDialog" name="DialogBC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Bright and Contrast</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lab_contrast">
     <property name="text">
      <string>contrast</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="slid_contrast">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lab_brighter">
     <property name="text">
      <string>brightness</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="slid_brighter">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="chk_auto_cnb">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>auto</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
